src/app/features/ledger/steps/sign-ledger-transaction.layout.tsx

```diff
@@ -12,19 +12,22 @@ import { LedgerSuccessLabel } from '../components/success-label';
 
 interface TransactionDetailProps {
   children: React.ReactNode;
+  isFullPage: boolean;
   title: string;
   tooltipLabel?: string;
 }
 function TransactionDetail(props: TransactionDetailProps) {
+  const { children, isFullPage, title, tooltipLabel } = props;
+
   return (
     <Flex borderColor="#DCDDE2" flexDirection="column">
-      <Caption>{props.title}</Caption>
+      <Caption>{title}</Caption>
       <Flex alignItems="center" mt="base">
-        <Text overflowWrap="break-word" maxWidth="360px">
-          {props.children}
+        <Text overflowWrap="break-word" maxWidth={['280px', '360px']}>
+          {children}
         </Text>
-        {props.tooltipLabel ? (
-          <Tooltip label={props.tooltipLabel} maxWidth="260px" placement="right">
+        {tooltipLabel ? (
+          <Tooltip label={tooltipLabel} maxWidth={isFullPage ? '260px' : '210px'} placement="right">
```

Comment:
This prop doesn't accept the responsive array so I had to use the `useMediaQuery`.